src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -422,4 +445,35 @@ function AppleHealthMenuItem(props: AppleHealthMenuItemProps) {
             </div>
         }
     </div>;
+}
+
+function WeatherMenuItem(props: { settings: DataCollectionSettings | null, participantInfo: ParticipantInfo | null, postalCodeSurveyName: string }){
+    let action = () => {
+        MyDataHelps.startSurvey(props.postalCodeSurveyName);
+    }
+
+    const postalCodeCustomFields = Object.keys(props.participantInfo ? props.participantInfo.customFields : {}).filter( (k) => k.endsWith('PostalCode'));
+    const postalCodes = [...postalCodeCustomFields.map((cf) => props.participantInfo?.customFields[cf]), props.participantInfo?.demographics.postalCode];
```

Comment:
Minor:  You can drop the parens from these single arg arrow functions if you'd like.